src/components/sidebar/cart/Cart.js

```diff
@@ -46,7 +53,23 @@ function Cart({ classes }) {
           {' '}
           {label}
         </Typography>
-        {!!uniqueSections.length && <Button onClick={handleClick}> Remove All </Button>}
+        {!!uniqueSections.length && <Button onClick={handleClickOpen}> Remove All </Button>}
+        <Dialog
```

Comment:
The Dialog should probably be its own separate component

Reply:
I was thinking about that as well, but not sure what should I name it and in which folder should I put this component.

Reply:
It can live in the same directory - I'll let you come up with a descriptive and succinct name. Naming things is hard, but only way to get better is with practice! 😄